ts/src/pro/bittrex.ts

```diff
@@ -632,30 +637,24 @@ export default class bittrex extends bittrexRest {
         //     7. Continue to apply messages as they are received from the socket as long as sequence number on the stream is always increasing by 1 each message (Note: for private streams, the sequence number is scoped to a single account or subaccount).
         //     8. If a message is received that is not the next in order, return to step 2 in this process
         //
-        const orderbook = await this.subscribeToOrderBook (negotiation, symbol, limit, params);
-        return orderbook.limit ();
-    }
-
-    async subscribeToOrderBook (negotiation, symbol, limit: Int = undefined, params = {}) {
```

Comment:
this separate method existed only in bittrex and was odd. all other exchagnes have the subscription exec lines inside `watchOrderBook` itself, no dedicated `subscribeToOrderBook` method needed.